gen3-integration-tests/tests/test_indexing_page.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+import requests
+import os
+import utils.gen3_admin_tasks as gat
+
+from utils import logger
+from gen3.auth import Gen3Auth
+from gen3.index import Gen3Index
+from pages.login import LoginPage
+from pages.indexing_page import IndexingPage
+from services.indexd import Indexd
+
+
+@pytest.mark.portal
+@pytest.mark.sower
+@pytest.mark.ssjdispatcher
+class TestIndexingPage:
+    variables = {}
+
+    def setup_class(cls):
+        cls.variables["test_guid"] = "c2da639f-aa25-4c4d-8e89-02a143788268"
```

Comment:
do we need a fixed value here?

Reply:
The GUID is mentioned in the valid_test_manifest.tsv file in test_data folder. I am just adding it here for validation of expected_result.